sfdx/README.md

```diff
@@ -16,19 +16,19 @@ A plugin for the Salesforce CLI to create mobile applications to interface with
 
 ### Install as plugin
 
-1. Install plugin: `sfdx plugins:install sfdx-mobilesdk-plugin`
+1. Install the plugin: `sfdx plugins:install sfdx-mobilesdk-plugin`
 
 ## Help
 ```
 -> sfdx mobilesdk --help
 Usage: sfdx COMMAND [command-specific-options]
 
-Help topics, type "sfdx help TOPIC" for more details:
+Help topics. Type "sfdx help TOPIC" for more details.
 
-  sfdx mobilesdk:ios         # command for building an iOS native mobile application using Saleforce Mobile SDK
-  sfdx mobilesdk:android     # command for building a Android native mobile application using Saleforce Mobile SDK
-  sfdx mobilesdk:hybrid      # command for building an hybrid mobile application using Saleforce Mobile SDK
-  sfdx mobilesdk:reactnative # command for building a React Native mobile application using Saleforce Mobile SDK
+  sfdx mobilesdk:ios         # Command for building an iOS native mobile application using Saleforce Mobile SDK
```

Comment:
It mirrored the force plugin and it's all lower case there
```
-> sfdx force --help
Usage: sfdx COMMAND [command-specific-options]

Help topics, type "sfdx help TOPIC" for more details:

  sfdx force:alias       # manage username aliases
  sfdx force:apex        # work with apex code
  sfdx force:auth        # authorize an org for use with the cli
  sfdx force:config      # configure the salesforce cli
...
```

Reply:
Also no periods at the end.

Reply:
I don't know why it needs to match their help strings. The periods were my own arbitrary style decision and aren't required here, so I don't mind dropping those. With capitalization, though, I think we can (and should) do better.